packages/shorebird_cli/lib/src/executables/devicectl/apple_device.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:pub_semver/pub_semver.dart';
+import 'package:shorebird_cli/src/extensions/version.dart';
+
+part 'apple_device.g.dart';
+
+/// {@template apple_device}
+/// A pared-down version of a CoreDevice, as represented in the JSON output of
+/// `xcrun devicectl list devices`.
```

Comment:
I assume there's no link to a schema/docs we could link to? 🤣

Reply:
If only. `devicectl` seems to be entirely undocumented. The flutter_tool comments simply reference json output :(